core/extension/gdextension_interface.cpp

```diff
@@ -992,6 +992,18 @@ static GDExtensionObjectPtr gdextension_object_get_instance_from_id(GDObjectInst
 	return (GDExtensionObjectPtr)ObjectDB::get_instance(ObjectID(p_instance_id));
 }
 
+static GDExtensionBool gdextension_object_get_class_name(GDExtensionConstObjectPtr p_object, GDExtensionClassLibraryPtr p_library, GDExtensionStringNamePtr r_class_name) {
+	if (!p_object) {
+		return false;
+	}
+	Object *o = (Object *)p_object;
```

Comment:
I think it's not necessary here to cast away `const` from the pointer, because `get_class_name_for_extension()` is a const method, and we have `GDExtensionConstObjectPtr` for the parameter (as opposed to `GDExtensionObjectPtr`).

This line could be written as
```cpp
const Object *o = reinterpret_cast<const Object*>(p_object);
```

or, if you want to make it consistent with other functions:
```cpp
const Object *o = (const Object *)p_object;
```

Reply:
Ah, good catch, thanks! It's hard to keep track of `const` when the compiler can't yell at me :-)